products/opensuse/main.pm

```diff
@@ -391,6 +391,7 @@ sub load_consoletests() {
             loadtest "console/http_srv.pm";
             loadtest "console/mysql_srv.pm";
             loadtest "console/dns_srv.pm";
+            loadtest "console/openvswitch.pm";
             if (check_var('ARCH', 'x86_64')) {
                 loadtest "console/docker.pm";
```

Comment:
That's why I put the openvswitch where docker was :) I though that if docker is OK to run on that context, then openvswitch is OK too.

I have a question though. Do you mean to add the test in ```load_extra_tests```? This has the following condition

```return !get_var("INSTALLONLY") && get_var("DESKTOP") !~ /textmode/ && !get_var("DUALBOOT") && !get_var("RESCUECD");```

so if I read this correctly, the test will not run if ```DESKTOP == textmode``` which is slightly weird because afaik ```textmode``` is meant for server-like installations where openvswitch is more likely to be there than in a regular desktop environment.